### PR TITLE
Adding Photostructure and Documentation

### LINKED ISSUE #1087: New App: PhotoStructure
# Feature request

**Is your feature request related to a problem? Please describe.**

Nope. Just looking to have a new app included in DockSTARTerif possible.

**Describe the solution you'd like**

As a user of DockSTARTer, I want to be able to install Photostructure via the DockSTARTer interface, so that I can use Photostructure on my server.

https://photostructure.com/server/photostructure-for-docker-compose/

**Describe alternatives you've considered**

Doing the override myself. I can't figure it out :(

**Additional context**

I think this would be a great alternative to Piwigo!

### PR DESCRIPTION
**Pull request by angrycuban13**
# Pull request

**Purpose**
Closes #1087 

**Approach**
How does this change address the problem?

**Open Questions and Pre-Merge TODOs**
Check all boxes as they are completed

- [x] Use github checklists. When solved, check the box and explain the answer.

**Learning**
Describe the research stage
Links to blog posts, patterns, libraries or addons used to solve this problem

**Requirements**
Check all boxes as they are completed

- [x] These changes meet the standards for [contributing](https://github.com/GhostWriters/DockSTARTer/blob/master/.github/CONTRIBUTING.md).
- [x] I have read the [code of conduct](https://github.com/GhostWriters/DockSTARTer/blob/master/.github/CODE_OF_CONDUCT.md).

### CODE REVIEW

File: compose/.apps/photostructure/photostructure.labels.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+services:
+  photostructure:
+    labels:
+      com.dockstarter.appinfo.description: "Photostructure is your new home for all your photos & videos."
+      com.dockstarter.appinfo.deprecated: "false"
+      com.dockstarter.appinfo.nicename: "photostructure"
```
Comment (nemchik) [inline compose/.apps/photostructure/photostructure.labels.yml:6]: Caps?

Comment (angrycuban13) [reply compose/.apps/photostructure/photostructure.labels.yml:6]: We need to fix a few then 😝

Comment (angrycuban13) [reply compose/.apps/photostructure/photostructure.labels.yml]: Fixed in ee33a9e

Comment (nemchik) [reply compose/.apps/photostructure/photostructure.labels.yml]: > We need to fix a few then 😝

Yeah, most of them are set right but i'm sure theres some in here that could have some caps. I tried to use whatever the official app uses minus special characters and spaces.

Comment (angrycuban13) [reply compose/.apps/photostructure/photostructure.labels.yml]: a3f7301 for this one and #1090 for the rest of the apps 👍

**Review by nemchik — APPROVED** [review]

**Pull request merged**